examples/README-CN.md

```diff
@@ -18,6 +18,12 @@ CDK 使用代码的示例和最佳实践分类如下：
 
 ### 产品文档
 
+#### ecs
+
+| 文档链接                                                                | 项目目录                                       | 支持语言                                                |
+|-----------------------------------------------------------------------------|----------------------------------------------------------|----------------------------------------------------------|
+| [部署和使用SVN](https://help.aliyun.com/zh/ecs/use-cases/deploying-and-using-svn)|[usage-exmaples](./product-documents/ecs/usage-examples/) | [python](./product-documents/ecs/usage-examples/python/) |
```

Comment:
The project directory name should be a summary of the document name with hyphen. Are you sure you want usage-examples?

Reply:
Updated the file basing on the new format.